go/internal/feast/featurestore.go

```diff
@@ -297,6 +315,10 @@ func (fs *FeatureStore) readFromOnlineStore(ctx context.Context, entityRows []*p
 	requestedFeatureViewNames []string,
 	requestedFeatureNames []string,
 ) ([][]onlinestore.FeatureData, error) {
+	// Create a Datadog span from context
```

Comment:
I think these can be removed or updated as datadog is not part of feast

Reply:
Yea. Datadog needs to be replaced with OTEL.

Reply:
yes, I will replace them with OTEL after this PR get merged.